src/wazuh_modules/vulnerability_scanner/CMakeLists.txt

```diff
@@ -68,3 +71,5 @@ else()
   #add_subdirectory(testtool)
 endif(UNIT_TEST)
 
+#add_subdirectory(testtool)
+#add_subdirectory(tests)
```

Comment:
Please, check these lines.

Reply:
Good catch, probably some rebase caused that. Removed those lines.